examples/petsc/setup.h

```diff
@@ -461,7 +460,7 @@ static int CreateRestrictionPlex(Ceed ceed, CeedInterlaceMode imode, CeedInt P,
       }
       // Essential boundary conditions are encoded as -(loc+1)
       PetscInt loc = indices[i] >= 0 ? indices[i] : -(indices[i] + 1);
-      erestrict[eoffset++] = loc/ncomp;
+      erestrict[eoffset++] = loc;
```

Comment:
As an extra win, this function starts looking more straightforward with this change.